src/services/routingSlip.services.ts

```diff
@@ -27,6 +28,20 @@ export default class RoutingSlip {
     )
   }
 
+  public static async adjustRoutingSlip (
+    routingSlipRequest: Array<object>,
```

Comment:
`Array<object>` might be worth creating a typescript interface for the request.. it might already exist infact.

Reply:
yes, found Payment interface, so I changed Array<object> to Payment[]